keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -888,20 +896,20 @@ class ClusterHandler(
   /**
    * Translates a ServerGroup object to a ClusterSpec.ServerGroupSpec with default values omitted for export.
    */
-  private fun ServerGroup.exportSpec(account: String, application: String): ClusterSpec.ServerGroupSpec {
-    val defaults = ClusterSpec.ServerGroupSpec(
+  private fun ServerGroup.exportSpec(account: String, application: String): ServerGroupSpec {
+    val defaults = ServerGroupSpec(
       capacity = Capacity(1, 1, 1),
       dependencies = ClusterDependencies(),
-      health = Health().exportSpec(),
+      health = Health().toSpecWithoutDefaults(),
       scaling = Scaling(),
       tags = emptyMap()
     )
 
-    val thisSpec = ClusterSpec.ServerGroupSpec(
-      launchConfiguration = launchConfiguration.exportSpec(account, application),
+    val thisSpec = ServerGroupSpec(
+      launchConfiguration = launchConfiguration.exportSpec(account, location.region, application, launchConfiguration.instanceType),
```

Comment:
we need to pass in the instance type here, otherwise we incorrectly think it is overridden.